src/main/java/juja/sqlcmd/InMemoryDatabaseManager.java

```diff
@@ -83,6 +84,24 @@ boolean insert(DataSet row) {
             return true;
         }
 
+        boolean delete(int id) {
+            int rowIndexWithId = findRowById(id);
+            if (rowIndexWithId != -1) {
+                rows.remove(rowIndexWithId);
+                return true;
+            }
+            return false;
+        }
+
+        private int findRowById(int id) {
+            for (int index = 0; index < rows.size(); index++) {
+                DataSet currentRow = rows.get(index);
+                String currentRowId = currentRow.values()[0];
+                if (currentRowId.equals(String.valueOf(id))) return index;
```

Comment:
It's better to use braces here  to reduce the possibility of confusion when reading or modifying the code. And also that is the part of google code style convesion - http://bit.ly/2FUBU7N

Reply:
Agree with Shtramak

Reply:
Agree with Shtramak